### DISCUSSION TITLE
pass legacy resource resource as a dependency to ConfigurableResource

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18371] [Q&A]
Hi folks!

I'm in process of transition from legacy Resources to Pydantic-resources and wonder is there a way to pass legacy resource resource as a dependency to ConfigurableResource?
<https://docs.dagster.io/concepts/resources#resources-that-depend-on-other-resources|Documentation> has an example of Pydantic-to-Pydantic dependency, but it's completely unclear if I can pass old-style resource there.

I understand that mixing two resource styles is not the best idea overall, but basically has no choice since moving all existing code base to Pydantic at once is quite a time-investment for our codebase.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701264212410149?thread_ts=1701264212.410149&cid=C01U954MEER).*

**Comment by smackesey** [comment]
cc @benpankow I'm unclear on the answer here. Also I think that the "resources that depend on other resources" section needs a note on this regardless of what the answer is.
